marconi-mamba/test/Spec/Marconi/Mamba/Api/Query/Indexers/Utxo.hs

```diff
@@ -136,3 +158,82 @@ generateUtxoRowJson = do
                 (Just $ C.hashScript script)
         utxoRow = UtxoRow utxo (C.SlotNo 1) blockHeaderHash
     pure $ Aeson.encodePretty utxoRow
+
+-- | Start a servant process and execute the client action
+withUserApp
```

Comment:
Maybe a more significant name? If it wasn't for the comment, the name didn't mean anything. Don't have a good one though..

Reply:
changed to `withMarconiHttpApp`